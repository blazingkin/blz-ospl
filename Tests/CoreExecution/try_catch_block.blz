:main
	try
		1 / 0
		print("unreachable!")
	catch error
		print("fell into catch block")
	end
	print("finished catch block")
end
