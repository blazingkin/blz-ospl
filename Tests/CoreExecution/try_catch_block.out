fell into catch block
finished catch block
